modules/core/src/test/java/org/apache/ignite/testframework/junits/common/GridCommonAbstractTest.java

```diff
@@ -2810,4 +2815,35 @@ protected DistributedChangeableProperty<Serializable> computeJobWorkerInterruptT
 
         return timeoutProperty;
     }
+
+    /** Clean work directory, excluding snapshot directory. */
+    protected IgniteEx restartWithCleanPersistence(int nodes, Collection<String> caches) throws Exception {
+        stopAllGrids();
```

Comment:
It seems to be clearer given the method name if we use ` int nodesCnt = Ignition.allGrids().size();` instead of `nodes` argument.

That is up to you.

Reply:
I also thought this way. But it's possible that user might have some client nodes in random order. Then we should filter G.allGrids() for server nodes and client nodes, and re-start only server nodes. Looks too complicated actually.

Then I think it's ok to specify nodes param for a while. Somebody who wants to use this method in the future can change it.

Reply:
OK.

Reply:
Remove this function at all. Reuse `cleanPersistenceDir` instead.